src/screens/CourseScreen/index.tsx

```diff
@@ -29,6 +34,8 @@ interface CommentObj {
 	content: string;
 }
 export const CourseScreen = () => {
+	const [openModal, setOpenModal] = useState(false);
+	const [feedback, setFeedBack] = useState('');

```

Comment:
we don't need the state in this component the feedback state will only live in the deefback modal so delete it and delete the feedback modal state props and include the state in the feedback modal